Sources/Momento/CacheClient.swift

```diff
@@ -5,14 +5,16 @@ public enum ScalarType {
     case data(Data)
 }
 
-typealias CacheClientProtocol = ControlClientProtocol & DataClientProtocol
+// The CacheClient interface provides user-friendly, public-facing methods
+// with default values for non-required parameters and overloaded functions
+// that accept String and Data values directly rather than ScalarTypes
 
 /**
  Client to perform operations against Momento Serverless Cache.
  To learn more, see the [Momento Cache developer documentation](https://docs.momentohq.com/cache)
  */
 @available(macOS 10.15, iOS 13, *)
-public class CacheClient: CacheClientProtocol {
+public class CacheClient {
```

Comment:
I don't really see an advantage in doing this. I kind of like having the protocols in place like you had them to really enforce that we have everything hooked up correctly. I also think you could probably use the protocol extensions to do a lot of the type normalization you'll need to DRY out the implementations of the overloaded methods in the `DataClient`.